haystack/nodes/file_converter/tika.py

```diff
@@ -93,12 +92,16 @@ def __init__(
             attributes. If you want to ensure you don't have duplicate documents in your DocumentStore but texts are
             not unique, you can modify the metadata and pass e.g. `"meta"` to this field (e.g. [`"content"`, `"meta"`]).
             In this case the id will be generated by using the content and the defined metadata.
+        :param timeout: How many seconds to wait for the server to send data before giving up,
+            as a float, or a :ref:`(connect timeout, read timeout) <timeouts>` tuple.
+            Defaults to 10 seconds.
+        :type timeout: Union[float, Tuple[float, float]]
```

Comment:
No need to add the type here as we already have it in the type annotation in the methods signature